sleep_avg is no longer a meaningful value being non linear and scheduler
dependant. Remove it from proc.

Signed-off-by: Con Kolivas <kernel@kolivas.org>


Index: linux-2.6.10-rc1-mm4/fs/proc/array.c
===================================================================
--- linux-2.6.10-rc1-mm4.orig/fs/proc/array.c	2004-11-10 09:17:06.263256565 +1100
+++ linux-2.6.10-rc1-mm4/fs/proc/array.c	2004-11-10 09:23:23.433661974 +1100
@@ -163,7 +163,6 @@ static inline char * task_state(struct t
 	read_lock(&tasklist_lock);
 	buffer += sprintf(buffer,
 		"State:\t%s\n"
-		"SleepAVG:\t%lu%%\n"
 		"Tgid:\t%d\n"
 		"Pid:\t%d\n"
 		"PPid:\t%d\n"
@@ -171,7 +170,6 @@ static inline char * task_state(struct t
 		"Uid:\t%d\t%d\t%d\t%d\n"
 		"Gid:\t%d\t%d\t%d\t%d\n",
 		get_task_state(p),
-		(p->sleep_avg/1024)*100/(1020000000/1024),
 	       	p->tgid,
 		p->pid, p->pid ? p->group_leader->real_parent->tgid : 0,
 		p->pid && p->ptrace ? p->parent->pid : 0,
